packages/core/src/common/errors.ts

```diff
@@ -120,3 +120,11 @@ export const OVERLAY_WITH_MULTIPLE_CHILDREN_REQUIRES_CHILD_REFS =
     ns + ` <Overlay2> requires childRefs prop when rendering multiple child elements`;
 export const OVERLAY_CHILD_REQUIRES_KEY =
     ns + ` <Overlay2> requires each child element to have a unique key prop when childRefs is used`;
+
+// prop deprecation warnings
```

Comment:
There will be quite a few other warnings related to prop deprecations in upcoming PRs, so creating a new section to house them here. https://github.com/palantir/blueprint/blob/61b79df070ba3e432bce2ec8a3d42ccca4f37aed/packages/core/src/common/errors.ts#L130-L151